File: Week_4/14.5_problem_2-loop_practice_2.c
// Problem_2 Question: Take a positive integer N as input and print the pattern shown in the sample input output.

/* 
sample input: 5

output:
*********
 *******
  *****
   ***
    *

 */


#include <stdio.h>
#include <limits.h>
#include <string.h>
#include <math.h>

int main()
{
    int n;
    scanf("%d", &n);

    int s = 0;
    int t = (n * 2) - 1;

    for (int i = 1; i <= n; i++)
    {
        for (int j = 1; j <= s; j++)
        {
            printf(" ");
        }
        for (int k = 1; k <= t; k++)
        {
            printf("*");
        }
        s++;
        t -= 2;
        printf("\n");
    }

    return 0;
}

File: Week_4/14.5_problem_2-loop_practice_2.py
# Problem_2 Question: Take a positive integer N as input and print the pattern shown in the sample input output.

""" sample input: 5

output:
*********
 *******
  *****
   ***
    * 
    
"""


n = int(input())

s = 0
t = (n * 2) - 1

for i in range(1, n + 1):
    for j in range(1, s + 1):
        print(" ", end="")
    for k in range(1, t + 1):
        print("*", end="")

    s += 1
    t -= 2
    
    print()
